cuqi/samples/_samples.py

```diff
@@ -92,7 +92,15 @@ def parameters(self):
         else:
             vals = self
         return type(self)(vals,is_par=True,geometry=self.geometry)
-    
+
+    def to_numpy(self):
```

Comment:
This is a helper function that access the numpy array in CUQIarray. I found it a bit indirect how to return the numpy array itself from the CUQIarray so I added this method. It could be useful for the user. I do not think self.array works to retrieve the numpy array.

I added some documentation to clarify the effect of is_par.